.github/workflows/buildAndTest.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Set up JDK
       uses: actions/setup-java@v2
       with:
-        java-version: 8
+        java-version: 11
```

Comment:
To note: this PR will already (partially) work towards the migration of a minimum requirement of Java 11, should therefore probably be merged together with a clean implementation for #100 
@Eddykasp do you remember why this was upped here already?

Reply:
This must come from my changes in #137